restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -288,25 +317,6 @@ public void tableExists(String keyspaceName, String tableName) {
     }
   }
 
-  private Query getInsertQueryForTable(ImmutableKeyspaceAndTable info) {
-    // Add a bind position for each column
-    List<ValueModifier> markers = new ArrayList<>(allColumnNames.size());
-    for (String columnName : allColumnNames) {
-      markers.add(ValueModifier.marker(columnName));
-    }
-
-    // Add a bind position for a timestamp, build, and return
-    return dataStore
-        .prepare(
-            dataStore
-                .queryBuilder()
-                .insertInto(info.getKeyspace(), info.getTable())
-                .value(markers)
-                .timestamp()
-                .build())
-        .join();
-  }
-
```

Comment:
good catch! This is exactly what I am talking about, removing peace by peace from the `DocumentDB`.. For V2 we need to do this..